src/bcs/index.ts

```diff
@@ -1,5 +1,10 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-export * from "./serializer";
+export * from "./serializable/fixed-bytes";
+export * from "./serializable/move-primitives";
+export * from "./serializable/move-structs";
+
+export * from "./serializable/entry-function-bytes";
```

Comment:
why each export?

Reply:
Oops, think that was from when I was testing things to avoid circular deps